Sources/SwiftDocC/Indexing/Navigator/NavigatorIndex.swift

```diff
@@ -358,7 +358,20 @@ public class NavigatorIndex {
             default: self = .article
             }
         }
-        
+
+        /// Whether this page kind references a symbol.
+        var isSymbolKind: Bool {
+            // This switch is written with the non-symbol kinds written explicitly on the assumption
+            // that new symbol kinds will be added over time, but that non-symbol kinds are much
+            // more rare to add.
+            switch self {
+            case .root, .article, .tutorial, .section, .learn, .overview, .resources, .framework,
+                    .buildSetting, .sampleCode, .languageGroup, .container, .groupMarker:
```

Comment:
What are containers? The only other reference to containers in the file that I can find is "containersymbol", which makes it sounds like a container is a symbol.

Reply:
To be honest, i'm not sure. My assumption is that `container` here references a kind of API Collection, but i'll defer to @d-ronnqvist in case he knows more.

Reply:
I don't recall. Maybe @franklinsch remembers.